### DISCUSSION TITLE
Katana Installation

### DISCUSSION THREAD

**Discussion by hammad-nv** [projectdiscovery/katana#498] [Q&A]
Environment 
Mac M1
MacOS: Ventura

Go: `go version go1.20.5 darwin/arm64`

- After installing katana via command `go install github.com/projectdiscovery/katana/cmd/katana@latest`
- And then running command `katana -h` I got this
   `zsh: command not found: katana`

**Comment by MetzinAround** [comment]
Can you check your PATH? it's possibly you need to add your gopath to your PATH. 

https://www.youtube.com/watch?v=wtHcXA2l9c4 talks about the process.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
